From 1a694da3421498c4c90986aa5f4b34d868290c9d Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sun, 16 Oct 2016 23:57:43 -0700
Subject: [PATCH 08/10] Wifi: Quiet some excessive debug output

* Specify LOG_NDEBUG for eng builds instead of userdebug
* Don't use LOGE for non-fatal errors stemming from driver feature
  support.

Change-Id: I15130e3700cb0608c9da09d05304f7aebf441c61
---
 qcwcn/wifi_hal/Android.mk  | 2 +-
 qcwcn/wifi_hal/llstats.cpp | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/qcwcn/wifi_hal/Android.mk b/qcwcn/wifi_hal/Android.mk
index 5500db1..643b7e8 100644
--- a/qcwcn/wifi_hal/Android.mk
+++ b/qcwcn/wifi_hal/Android.mk
@@ -19,7 +19,7 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_CFLAGS := -Wno-unused-parameter
-ifeq ($(TARGET_BUILD_VARIANT),userdebug)
+ifeq ($(TARGET_BUILD_VARIANT),eng)
 LOCAL_CFLAGS += "-DLOG_NDEBUG=0"
 endif
 
diff --git a/qcwcn/wifi_hal/llstats.cpp b/qcwcn/wifi_hal/llstats.cpp
index e12b61f..ed8af1c 100644
--- a/qcwcn/wifi_hal/llstats.cpp
+++ b/qcwcn/wifi_hal/llstats.cpp
@@ -934,7 +934,7 @@ int LLStatsCommand::handleResponse(WifiEvent &reply)
                 {
                     if (!tb_vendor[QCA_WLAN_VENDOR_ATTR_LL_STATS_NUM_RADIOS])
                     {
-                        ALOGE("%s: QCA_WLAN_VENDOR_ATTR_LL_STATS_NUM_RADIOS"
+                        ALOGD("%s: QCA_WLAN_VENDOR_ATTR_LL_STATS_NUM_RADIOS"
                               " not found", __FUNCTION__);
                         return WIFI_ERROR_INVALID_ARGS;
                     }
@@ -945,7 +945,7 @@ int LLStatsCommand::handleResponse(WifiEvent &reply)
                         QCA_WLAN_VENDOR_ATTR_LL_STATS_RADIO_NUM_CHANNELS
                         ])
                     {
-                        ALOGE("%s:"
+                        ALOGD("%s:"
                             "QCA_WLAN_VENDOR_ATTR_LL_STATS_RADIO_NUM_CHANNELS"
                             " not found", __FUNCTION__);
                         status = WIFI_ERROR_INVALID_ARGS;
-- 
2.17.1

